src/views/Profile/AdminTable/AdminDatePicker.tsx

```diff
@@ -32,18 +35,100 @@ interface IMaterialsDate {
   setChanges: (...params: any[]) => void;
 }
 
+interface ISelectedDay {
+  position: string;
+  bgColor: string;
+  value: string;
+}
+
+const datePickerReducer = (
+  state: {
+    firstSelectedDay: ISelectedDay;
+    secondSelectedDay: ISelectedDay;
+  },
+  action: { type: string; payload: ISelectedDay },
```

Comment:
it always makes sense to type `type` part. then TS will save our backs when we make a typo in `type` value